project_report/tex/requirements.tex

```diff
@@ -9,7 +9,13 @@ \subsubsection{Functional \& Non-Functional Requirements}
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
 \subsubsection{The Simulator}
-
+The group are going to use the simulator software to test the software in the
+development cycle.
+The simulator is not a replacement of the machine since there is only
+so much randomness and correctness you can get out of a simulator, but it is 
+still very important to have the simulator, for making prototypes and make 
+multiple tests for it, so the group don't push any regressions to the production
```

Comment:
for = on
don't = doesn't

What do u mean with "regressions"?

Reply:
Like we make some bug, without realizing, the test and the simulator should get that.

Reply:
get it like catch it before it is hitting production